.github/ISSUE_TEMPLATE/Issue.yml

```diff
@@ -0,0 +1,18 @@
+name: General Issue
+description: Your issue does not fit the categories already mentioned.
+title: " "
+labels: 
+  - help wanted
+body:
+  - type: markdown
+    attributes:
+      value: >
+        If you want to discuss more about the issue, reach us at our [Slack Channel](https://app.slack.com/client/T0QK6T7A7/C0QK9ARNW)
```

Comment:
Is this the correct url?

Reply:
I obtained this from the general channel, but I believe it should be replaced with the main [slack-redirecting](https://stingraysoftware.slack.com/ssb/redirect) link, thank you for spotting this!

Reply:
The link is this: https://join.slack.com/t/stingraysoftware/shared_invite/zt-49kv4kba-mD1Y~s~rlrOOmvqM7mZugQ

Reply:
> I obtained this from the general channel, but I believe it should be replaced with the main [slack-redirecting](https://stingraysoftware.slack.com/ssb/redirect) link, thank you for spotting this!

this was on the GSOC page, sorry for the oversight!